docs/proposed/psa-driver-developer-guide.md

```diff
@@ -39,7 +39,8 @@ Mbed TLS calls driver entry points [as specified in the PSA Cryptography Driver
 ## Mbed TLS extensions
 
 The driver description can include Mbed TLS extensions (marked by the namespace "mbedtls"). Mbed TLS extensions are meant to extend/help integrating the driver into the library's infrastructure.
-* `"mbedtls/h_condition"` (optional, string) can include complex preprocessor definitions to conditionally include header files for a given driver. 
+
```

Comment:
Is this blank line intentional? In my markdown previewer (VSCode's built in one) it doesn't seem to make a difference?

Reply:
Depends on the markdown flavor. [CommonMark](https://spec.commonmark.org/0.30/#example-303) allows lists to start without a blank line above, but the original markdown doesn't.